File: c programs/Assignments/Assignment_3/Assign3_3.c
//Write a program which accept number from user and print even factors of that number. 
// i/p: 36
// o/p:  2  6  12  18

#include<stdio.h>

void DisplayEvenFactor(int iNo)
{
	
     int iCnt = 0;
	 
	 if(iNo <= 0)
	 {
		 iNo = -iNo;
	 }
	 for(iCnt = 1; iCnt<= iNo/2 ; iCnt++)
	 {
		 if((iNo % iCnt ==0) && (iCnt % 2 ==0))
		 {
			 printf("%d\t",iCnt);
		 }
	 }
	 
}
int main()
{
	int iValue = 0;
	
	printf("Enter the value\n");
	scanf("%d",&iValue);
	
	DisplayEvenFactor(iValue);
	
	return 0;
}

File: Python/Assignment_3/Assign3_3.py
#Write a program which accept N numbers from user and store it into List. Return Minimum
#number from that List.

#Input : Number of elements : 4
#Input Elements : 13 5 45 7
#Output : 5

def MIN(LIST):
  iMin = LIST[0]
  for i in LIST:                   #iteratr over list         
    if i < iMin:
	    iMin = i
  return iMin


def main():
 Arr = []
 print("Enter ele:")
 iSize = int(input())
 
 for i in range(iSize):
   print("INPUT ELEMENT IS:",i+1)
   iVlaue=int(input())
   Arr.append(iVlaue)
   
   print("Accept data:",Arr)
   iRet = MIN(Arr)
   
   print("MINIMUM NUMBER IS:~",iRet)
   
   
if __name__=="__main__":
  main()